### PR TITLE
Some backend stuff

### PR DESCRIPTION
**Pull request by aiecel**
Fixed some tiny issues

### CODE REVIEW

File: crap-advisor-api/src/main/kotlin/com/aiecel/crapadvisor/service/ReviewService.kt [hunk 1]
```diff
@@ -20,7 +20,11 @@ class ReviewService(
 
     private val log = KotlinLogging.logger { }
 
-    fun getByRestroomId(restroomId: Long) = reviewRepository.findAllByRestroomId(restroomId)
+    fun getByRestroomId(restroomId: Long): List<Review> {
+        val reviews = reviewRepository.findAllByRestroomId(restroomId)
+        log.debug("Fetched ${reviews.size} review(s) by restroom id $restroomId")
```
Comment (akichula) [inline crap-advisor-api/src/main/kotlin/com/aiecel/crapadvisor/service/ReviewService.kt:25]: You are really need this logs in master branch?

**Review by akichula — COMMENTED** [review]
@aiecel I left one comment, answer me and after discuss i can approve your merge request.

File: crap-advisor-api/src/main/kotlin/com/aiecel/crapadvisor/service/ReviewService.kt (hunk 1 above)
Comment (aiecel) [reply crap-advisor-api/src/main/kotlin/com/aiecel/crapadvisor/service/ReviewService.kt:25]: Man logs are very important, how can you see what the backend is doing when it's not writing in the logs? 
Yes, this stays

Comment (akichula) [reply crap-advisor-api/src/main/kotlin/com/aiecel/crapadvisor/service/ReviewService.kt:25]: Sounds convincing! I'll resolve this tread now

**Review by akichula — APPROVED** [review]
@aiecel Able to approve.

**Pull request merged**